### PR TITLE
[PVR] Addon API 5.10.0: New API function GetStreamReadChunkSize.

### PR DESCRIPTION
**Pull request by ksooo**
@FernetMenta as discussed.

I already adapted pvr.hts and setting chunk size to 64K solved all my buffering problems when playing recordings. ;-)

### CODE REVIEW

**Comment by ksooo** [comment]
@Jalle19 fyi

File: xbmc/pvr/addons/PVRClients.cpp [hunk 1]
```diff
@@ -559,6 +559,26 @@ std::string CPVRClients::GetBackendHostnameByClientId(int iClientId) const
...
+
+  if (item.HasPVRChannelInfoTag())
+    iClientID = item.GetPVRChannelInfoTag()->ClientID();
+  else if (item.HasPVRRecordingInfoTag())
+    iClientID = item.GetPVRRecordingInfoTag()->m_iClientId;
+
+  if (iClientID != PVR_INVALID_CLIENT_ID)
+  {
+    ForCreatedClient(__FUNCTION__, iClientID, [&iChunkSize](const CPVRClientPtr &client) {
+      return client->GetStreamReadChunkSize(iChunkSize);
```
Comment (FernetMenta) [inline xbmc/pvr/addons/PVRClients.cpp]: that returns PVR_ERROR and not chunkSize

Comment (FernetMenta) [reply xbmc/pvr/addons/PVRClients.cpp]: disregard :)

File: xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h [hunk 2]
```diff
@@ -519,6 +519,15 @@ extern "C"
    *  @remarks This will only be used if the backend doesn't provide a direct URL in the recording tag.
    */
   //@{
+  /*!
+   * Obtain the chunk size to use when reading streams.
```
Comment (FernetMenta) [inline xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h]: we should note that this has no effect when reading from demuxer

Comment (ksooo) [reply xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h]: Okay, will add this to the `@remarks` section for this function.

Comment (ksooo) [reply xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h]: done

**Review by FernetMenta — APPROVED** [review]

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
@wsnipex please update pvr addons PPA, thx.

**Pull request merged**